src/cabinetry/model_utils.py

```diff
@@ -318,68 +319,79 @@ def yield_stdev(
         # add total prediction (sum over samples)
         down_comb = np.vstack((down_comb, np.sum(down_comb, axis=0)))
         # turn into list of channels
-        down_yields = [
+        down_yields_per_channel = [
             down_comb[:, model.config.channel_slices[ch]]
             for ch in model.config.channels
         ]
-        # append list of yields summed per channel
-        down_yields += [
-            np.asarray(np.sum(chan_yields, axis=-1, keepdims=True))
-            for chan_yields in down_yields
-        ]
+
+        # calculate list of yields summed per channel
+        down_yields_sum = np.array(
+            [
+                np.sum(chan_yields, axis=-1, keepdims=True).tolist()
+                for chan_yields in down_yields_per_channel
+            ]
+        )
```

Comment:
Given that these are NumPy arrays, better to leave them as such
```suggestion
        down_yields_sum = np.stack(
            [
                np.sum(chan_yields, axis=-1, keepdims=True)
                for chan_yields in down_yields_per_channel
            ]
        )
```